src/Hive.Core/sia380/sia380.py

```diff
@@ -332,8 +346,20 @@ def main(room_properties, room_schedules, floor_area, T_e_hourly, T_i_ub_hourly,
 
     # Ventilation heat loss coefficient (Lüftungs-Wärmetransferkoeffizient), H_V
     # m3/h to m3/s, hence divide by 3600
-    H_V = Vdot_th / SECONDS_PER_HOUR * rho * c_p
-    H_V_no_heat_recovery = Vdot_th_no_heat_recovery / SECONDS_PER_HOUR * rho * c_p
+    H_V = (Vdot_th / SECONDS_PER_HOUR) * RHO * C_P
+    H_V_no_heat_recovery = (Vdot_th_no_heat_recovery / SECONDS_PER_HOUR) * RHO * C_P
+    
+    # Natural ventilation (no infiltration ? TODO)
+    if use_natural_ventilation and windows_count > 0:
+        # TODO picks the first window as the window to ventilate. Should be selected by user?
+        window_for_nat_vent_idx = next((i for i, s in enumerate(surface_type) if s == "transp"), None)
```

Comment:
why only one window? I would use all windows for nat vent

Reply:
the equation we use is for single-sided ventilation. If we want to do cross-ventilation (which makes more sense obv.), we need to use a slightly different equation that calculates the effective area of windows (which we can do i think with just surface area) but we also need the difference in height of the windows, which requires us knowing info about the Brep (average height of each window and change in those heights). Since we left that out for now, I am using single-sided ventilation. Maybe also why effects are minimal at the moment.

Areas calc looks something like this for the same room

![image](https://user-images.githubusercontent.com/53180223/125756951-6efb23b9-48a6-47a9-a34d-0a1803c798a0.png)

I think this can be resolved as part of #594

Reply:
Single-sided ventilation doesn't mean it is restricted to one window. Have a look at CibseAM10 p.45-46:
![image](https://user-images.githubusercontent.com/21927657/125835415-92b3e4ed-7295-4994-a763-bb7ace0d8146.png)
![image](https://user-images.githubusercontent.com/21927657/125835430-3075480e-d026-42a6-8681-6dd6eb2b03f2.png)
![image](https://user-images.githubusercontent.com/21927657/125835456-df66a0fc-df48-4734-b8f4-9cda67dd257d.png)

Just re-arrange for `$q$`, since `$A$` would be your total area of all windows.

But let's merge first and do this as part of #594

Reply:
Oh my mistake, I will address this in the other issue then and I use this as a reference.